e2e/test/scenarios/models/models-metadata.cy.spec.js

```diff
@@ -207,9 +207,11 @@ describe("scenarios > models metadata", () => {
     openQuestionActions();
     popover().findByTextEnsureVisible("Edit query definition").click();
 
-    cy.get(".ace_content").type(
-      "{selectAll}{backspace}SELECT TOTAL FROM ORDERS LIMIT 5",
-    );
+    cy.get(".ace_text-input")
+      .first()
```

Comment:
Does this element have anything more specific to pick it? 
This looks brittle and error-prone tbh.

Reply:
done